crates/bevy_input/src/gamepad.rs

```diff
@@ -2,6 +2,87 @@ use crate::{Axis, Input};
 use bevy_ecs::event::{EventReader, EventWriter};
 use bevy_ecs::system::{Res, ResMut, Resource};
 use bevy_utils::{tracing::info, HashMap, HashSet};
+use thiserror::Error;
+
+/// Errors that occur when setting axis settings for gamepad input.
+///
+/// + `LiveZoneLowerBoundOutOfRange(f32)`
+///
+///     The given parameter `livezone_lowerbound` was not in range -1.0..=0.0.
```

Comment:
you can put this docstring on the variant directly (same for all others)

Reply:
the second one

Reply:
Done.